modules/home-environment.nix

```diff
@@ -592,22 +592,37 @@ in
         ''
       else
         ''
+          function nixReplaceProfile() {
+            local oldNix="$(which nix)"
+
+            nix profile list \
+              | { grep 'home-manager-path$' || test $? = 1; } \
+              | awk -F ' ' '{ print $4 }' \
+              | cut -d ' ' -f 4 \
```

Comment:
Do you know why awk and cut are both used? To me they do the same thing, twice in a row and the cut call will actually not do anything

Reply:
Oh right, this looks wired, indeed. Thank you!

Reply:
fix in a follow up commit, since having this is result of a faithful copy of existing code.